Extension/src/nativeStrings.json

```diff
@@ -448,4 +448,4 @@
         "text": "Copying Declaration / Definition to clipboard failed: %s",
         "hint": "The operation 'Copy Declaration / Definition' on a function was not successful. %s is the error info that has a period at the end of the string."
     }
-}
\ No newline at end of file
+}
```

Comment:
I think the json formatter/linter is supposed add a newline at the end, but it was removed?